### PR TITLE
Feature: Using Citation Libraries

### PR DESCRIPTION
**Pull request by hbalenda**
Refactors the quire citation shortcode to use CSL. The shortcode uses Quire `page` or `publication` data, translates it into CSL-JSON ([CSL Docs](https://docs.citationstyles.org/en/stable/specification.html), [CSL-JSON Schema](https://github.com/citation-style-language/schema/blob/master/schemas/input/csl-citation.json)), and then uses [`simple-cite`](https://github.com/lewisacidic/simple-cite) and the styles `chicago-fullnote-bibliography` or `mla`.

Changes:
- Use citation-style packages to cite Quire pages and publication
- Add `citation` plugin to handle citation style and locale configuration
- Rename `publication.number_in_series` -> `publication.seriesNumber`
- Rename `publication.series` -> `publication.seriesTitle`

### CODE REVIEW

**Comment by geealbers** [comment]
@hbalenda, for the URL, in Hugo we started using `identifier.url` in publication,.yaml for the citations rather than `baseURL`, as we found there were many cases where baseURL wasn't set. Like for our netlify deploy.

**Comment by hbalenda** [comment]
@geealbers Ah good to know - we are using `config.baseURL` extensively in the eleventy project, but it makes sense that a user shouldn't be required to edit anything there. We can replace those references on a separate branch.

Also thank you for the review last week. I've changed the style to Chicago fullnote bibliography, added italics, and documented how to configure the citation style and locale. The formatting still has a couple (very) slight differences, though I think the library version is actually correct and that they are things that were broken when moving the citation shortcode to eleventy.

**Review by mphstudios — APPROVED** [review]

File: packages/11ty/_includes/components/citation/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Citation Component
+
+The citation component translates page data and data from `publication.yaml` into [CSL-JSON]((https://github.com/citation-style-language/schema/blob/master/schemas/input/csl-citation.json)) and uses the [Quire citation plugin](/blob/main/packages/11ty/_plugins/citations/README.md) to format that data in `MLA` or `Chicago` formats.
```
Comment (mphstudios) [inline packages/11ty/_includes/components/citation/README.md:3]:
```suggestion
The citation component translates page data and data from `publication.yaml` into [CSL-JSON]((https://github.com/citation-style-language/schema/blob/master/schemas/input/csl-citation.json)) and uses the [Quire citation plugin](/blob/main/packages/11ty/_plugins/citations/README.md) to render that data in `MLA` or `Chicago` formats.
```

File: packages/11ty/_plugins/citations/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+// .eleventy.js
+
+eleventyConfig.addPlugin(citationsPlugin, { 
+  styles: {
+    apa: require('style-apa')
+  }
+})
+```
+
+To use, set the citation shortcode `type` to the style name.
```
Comment (mphstudios) [inline packages/11ty/_plugins/citations/README.md:26]:
```suggestion
To use, set the `citation` shortcode `type` to the style name.
```

File: packages/11ty/_plugins/citations/README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# Quire Citations Plugin
+The Quire citations plugin uses [`simple-cite`]() and citation-styles [CSL-JSON](https://github.com/citation-style-language/schema#csl-json-schema) to generate citations with configurable locale and style. It registers an eleventy javascript function `formatCitation`, which is used to create page and publication citations from the citations component.
```
Comment (mphstudios) [inline packages/11ty/_plugins/citations/README.md:2]: inter-repository linking to the citation component woul
```suggestion
The Quire citations plugin uses [`simple-cite`](https://github.com/lewisacidic/simple-cite#readme) and citation-styles [CSL-JSON](https://github.com/citation-style-language/schema#csl-json-schema) to generate citations with configurable locale and style. It registers an eleventy javascript function `formatCitation`, which is used to create page and publication citations from the [`citations` component](https://github.com/thegetty/quire/packages/11ty/_includes/components/citation/).
```

**Review by mphstudios — CHANGES_REQUESTED** [review]
@hbalenda this is a really nice improvement to the use of standards and the flexibility of Quire.

**Review by mphstudios — APPROVED** [review]

File: packages/11ty/_plugins/citations/README.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+The Quire citations plugin uses [`simple-cite`](https://github.com/lewisacidic/simple-cite#readme) and citation-styles [CSL-JSON](https://github.com/citation-style-language/schema#csl-json-schema) to generate citations with configurable locale and style. It registers an eleventy javascript function `formatCitation`, which is used to create page and publication citations from the [`citation` component](https://github.com/thegetty/quire/packages/11ty/_includes/components/citation/).
+
+## Styles
+The default styles included with Quire are MLA and Chicago fullnote bibliography.
+
+### Change the style
+See the [`citation-styles` packages](https://github.com/lewisacidic/citation-styles/tree/master/packages) for a list of published styles.
+
+To add a new style, first install the package. 
+```
```
Comment (mphstudios) [inline packages/11ty/_plugins/citations/README.md:11]:
```suggestion
```sh
```

File: packages/11ty/_plugins/citations/README.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+npm install style-apa
+```
+
+Then set the style in `citationsPlugin.options`:
+```
```
Comment (mphstudios) [inline packages/11ty/_plugins/citations/README.md:16]:
```suggestion
```javascript
```

File: packages/11ty/_plugins/citations/README.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+eleventyConfig.addPlugin(citationsPlugin, { 
+  styles: {
+    apa: require('style-apa')
+  }
+})
+```
+
+To use, set the `citation` shortcode `type` to the style name.
+```
```
Comment (mphstudios) [inline packages/11ty/_plugins/citations/README.md:27]:
```suggestion
```javascript
```

File: packages/11ty/_plugins/citations/README.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+The default locale is `locale-en-us`. 
+
+### Set the locale
+See [citation-style-language](https://github.com/citation-style-language/locales) for a list of locales.
+
+To add a new locale, install it:
+```npm install locale-fr-fr```
+
+Then set the locale in `citationsPlugin.options`:
+```
```
Comment (mphstudios) [inline packages/11ty/_plugins/citations/README.md:41]:
```suggestion
```javascript
```

File: packages/11ty/_plugins/citations/README.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+## Locales
+The default locale is `locale-en-us`. 
+
+### Set the locale
+See [citation-style-language](https://github.com/citation-style-language/locales) for a list of locales.
+
+To add a new locale, install it:
+```npm install locale-fr-fr```
```
Comment (mphstudios) [inline packages/11ty/_plugins/citations/README.md:38]:
```suggestion
```sh
npm install locale-fr-fr
```
```

**Review by mphstudios — APPROVED** [review]

**Pull request merged**